files/classes/comment.py

```diff
@@ -230,12 +230,13 @@ def json_raw(self):
 			'is_pinned': self.is_pinned,
 			'distinguish_level': self.distinguish_level,
 			'post_id': self.post.id if self.post else 0,
-			'score': self.score,
-			'upvotes': self.upvotes,
-			'downvotes': self.downvotes,
 			'is_bot': self.is_bot,
-			'flags': flags,
```

Comment:
If we're not gonna send flags at all, might as well take out building a dict of them up on line 216 too (and ditto for submissions)